src/main/java/net/mcreator/element/types/Command.java

```diff
@@ -18,20 +18,27 @@
 
 package net.mcreator.element.types;
 
+import net.mcreator.blockly.data.BlocklyLoader;
+import net.mcreator.blockly.java.BlocklyToJava;
 import net.mcreator.element.GeneratableElement;
-import net.mcreator.element.parts.Procedure;
+import net.mcreator.generator.blockly.BlocklyBlockCodeGenerator;
+import net.mcreator.generator.blockly.ProceduralBlockCodeGenerator;
+import net.mcreator.generator.template.IAdditionalTemplateDataProvider;
 import net.mcreator.minecraft.MinecraftImageGenerator;
+import net.mcreator.ui.blockly.BlocklyEditorType;
 import net.mcreator.workspace.elements.ModElement;
 
+import javax.annotation.Nullable;
 import java.awt.image.BufferedImage;
+import java.util.Locale;
 
 @SuppressWarnings("unused") public class Command extends GeneratableElement {
 
 	public String commandName;
 
 	public String permissionLevel;
 
-	public Procedure onCommandExecuted;
+	public String argsxml;

```

Comment:
I'd remove empty lines between these 3 fields.

Reply:
They are ok, they split different categories.. it is our convention